src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -86,12 +86,12 @@ private void testCreateInstructorAction() throws Exception {
         BackDoor.deleteCourse(demoCourseId);
         BackDoor.deleteInstructor(demoCourseId, instructor.email);
         homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
If the retry manager just blindly runs tries whether they succeed or fail, then it's only a glorified for loop, right? I don't quite understand the `TestRunner` 😅